gravitee-plugin-annotation-processors/src/main/resources/templates/builderFooter.mustache

```diff
@@ -0,0 +1,18 @@
+
+        // Evaluate properties that needs EL, validate evaluatedConf and returns it
+        return Maybe
+            .merge(Flowable.fromIterable(toEval))
+            .ignoreElements()
+            .andThen(Completable.fromRunnable(() -> validateConfiguration(evaluatedConfiguration)))
+            .toSingle(() -> evaluatedConfiguration);
+    }
+
+    public Completable eval() {
+        return Completable.defer(() -> evalConfiguration()
+            .flatMapCompletable(evaluatedConfiguration ->
+                Completable.fromRunnable(() ->
+                    context.setInternalAttribute("{{evaluatedConfigurationName}}", evaluatedConfiguration)
```

Comment:
Knowing that the annotation could be used for configuring policies also and that policies can be put several times in a flow, having an attribute name hardcoded could lead to side effects (the second instance of the policy will get the configuration of the previous same policy).

Another option could be to generate an uuid and store it at the evaluator instance level and reuse it to store the evaluated configuration in the internal attributes without the risk to clash between policies.

Another benefit is that it becomes possible to store and retrieve the evaluated configuration pretty easily by always calling `xxxEvaluator.eval(ctx) `and checking internally if a corresponding evaluated config is already present in the internal context. That greatly simplifies the usage. WDYT?

Reply:
Yep I agree, I've made the changes.